### PR TITLE
Feature/add GitHub workflow

### PR DESCRIPTION
**Pull request by manuwei**
add github workflow

### CODE REVIEW

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# This workflow will build a Java project with Maven
```
Comment (manuwei) [inline .github/workflows/dockerhub.yml:1]: should the comments be adapted?

Comment (salmma) [reply .github/workflows/dockerhub.yml:1]: its just a comment you can simply remove it or adapt it for the ui

File: .github/workflows/dockerhub.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Build and push latest Pattern Atlas UI
+        if: ${{ steps.vars.outputs.tag == 'master' }}
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: planqk/qc-atlas-ui:latest
```
Comment (salmma) [inline .github/workflows/dockerhub.yml:36]: rename the image title for patternaltas

**Review by salmma — APPROVED** [review]
LG

**Review by salmma — CHANGES_REQUESTED** [review]
Last Lint check failed

**Review by salmma — APPROVED** [review]
LG

**Pull request merged**